src/content/self-hosted/install/releases.mdx

```diff
@@ -7,14 +7,22 @@ id: releases
 
 ## 1.15.0
 
-TBD
+22 November 2023
 
 This version is compatible with Kubernetes versions 1.25 to 1.27.
 
 ### Improvements
 
 - Upgrade ingress-nginx to 1.9.4 (Helm chart version 4.8.3)  <!-- 7386 -->
 - Enabled [chroot mode](https://kubernetes.io/blog/2022/04/28/ingress-nginx-1-2-0/) by default in both ingress-nginx controllers managed by Okteto. More info in [Upgrade your Okteto instance section](https://www.okteto.com/docs/self-hosted/install/upgrade/#upgrading-to-okteto-115x) <!-- 7425 -->
+- `okteto up` sessions are now garbage collected if no files have been synced for an hour   <!-- 7424 -->
+- Upgrade nginx to [1.8.4](https://github.com/kubernetes/ingress-nginx/releases/tag/controller-v1.9.4) (chart 4.8.3) <!-- 7386 -->
+
+### Bug Fixes
```

Comment:
I think there are 2 fixes missing:
* The one related to this commit `restore .gitmodules file after cloning`. I think this was generating some issue in the installer pods when there was submodules and images were being rebuilt always even if they were not needed cc @andreafalzetti could you provide more information?
* We have fix in the admin UI to show the proper configuration for the GC. cc @viddo Can provide more information

Reply:
I did include it in the original PR (7404), copy-paste that should have been included here:

> Fix wrong "Scale to Zero Period" value in the UI of Admin → Namespaces

Aside, figure you might want to look into why that wasn't extracted as expected?

Reply:
The fix I did on git submodules fixes the smart builds (or pull first) to work when git submodules is used and the `.gitmodules` is using `git`.

Related PR: https://github.com/okteto/app/pull/7406

For a release note I would write:

```
- Bug fix: smart builds work on repositories using `.gitmodules` containing submodules cloned via ssh key
```